pubspec.yaml

```diff
@@ -3,6 +3,10 @@ description: Small, easy to use and extensible logger which prints beautiful log
 version: 2.3.0
 repository: https://github.com/SourceHorizon/logger
 
+topics:
+- cli
+- logging
```

Comment:
It seems that `logging`, `logger` and `log` are the three most popular topics, how about we add all three?

Reply:
You can, but since they are all practically the same I landed a quick change to merge those topics into one:

https://github.com/dart-lang/pub-dev/blob/master/doc/topics.yaml#L65

But, by all means feel free to add whatever topics you think is best.
Also feel free to help merge topics you think are the same :)

Reply:
Note, the merging of the topics haven't taken effect yet.

It'll happen some time after next deployment. So maybe a week or two, typically.

Reply:
Thanks, I didn't know that topics can be merged!
In this case, disregard my comment, `logging` seems fine.